### PR TITLE
Update README for proto3

### PR DESCRIPTION
**Pull request by bkuehnert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +245,32 @@ Clear the value of a field. After the above call `(date-range.has-min-date
...
+}
+```
+
+The generated code for proto3 messages is similar to proto2 messages. The only
+difference is the introduction of fields with no specified label, which are
+known as "singular" fields. For singular fields, the state of being unset
+and the state of being set to the default value for the type are indistinguishable.
+So, `has-*` functions, such as `(event.has-day msg)` are not defined.
+
+The `has-*` functions for repeated fields are defined, and work as normal in proto3.
```
Comment (Slids) [inline README.md:269]: Not a huge fan of "work as normal", should explain normal.

File: README.md [hunk 2]
```diff
@@ -243,6 +245,32 @@ Clear the value of a field. After the above call `(date-range.has-min-date
...
+
+This library supports optional fields in proto3 messages. These fields have
+the same semantics as proto2 optional fields (namely, they have `has-*` functions).
```
Comment (Slids) [inline README.md:272]: Remove  (namely, they have `has-*` functions)?
Does it add anything?

File: README.md [hunk 3]
```diff
@@ -366,7 +394,8 @@ message Person {
   }
 }
 ```
-To access fields inside a oneof, just use the standard accessors outlined above. For example:
+To access fields inside a oneof, just use the standard accessors outlined above. These fields
```
Comment (Slids) [inline README.md:397]: remove just
In all honesty I probably would have written it to...

File: README.md (hunk 2 above)
Comment (bkuehnert) [reply README.md:272]: No, the fields are identical to proto2 optional fields in terms of generated code.

**Comment by cgay** [comment]
made these changes internally. closing.

**Pull request closed without merging**